src/test/java/org/stellar/sdk/OperationTest.java

```diff
@@ -69,7 +69,29 @@ public void testPaymentOperation() throws FormatException, IOException, AssetCod
 
         assertEquals(
                 "AAAAAQAAAAC7JAuE3XvquOnbsgv2SRztjuk4RoBVefQ0rlrFMMQvfAAAAAEAAAAA7eBSYbzcL5UKo7oXO24y1ckX+XuCtkDsyNHOp1n1bxAAAAAAAAAAAlQL5AA=",
-                operation.toXdrBase64());
+                operation.toXdrBase64(AccountConverter.enableMuxed()));
+    }
+
+    @Test
+    public void testMuxedPaymentOperation() throws FormatException, IOException, AssetCodeLengthInvalidException {
+        String source = "MA7QYNF7SOWQ3GLR2BGMZEHXAVIRZA4KVWLTJJFC7MGXUA74P7UJVAAAAAAAAAAAAAJLK";
+        String destination = "MDQNY3PBOJOKYZSRMK2S7LHHGWZIUISD4QORETLMXEWXBI7KFZZMKAAAAAAMV7V2XYGQO";
```

Comment:
We should have a test case mixing muxed and non-muxed addresses in the same operation.